common/changes/office-ui-fabric-react/tooltip_2019-02-07-23-13.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
We should remove this one since you have the other change file, right?

Reply:
Yea that seems correct